.github/workflows/main.yaml

```diff
@@ -73,8 +73,12 @@ jobs:
       - name: run python tests
         run: testing/python_test test_fot
       - name: setup fot stack
+        env:
+          FAUX_MAC_PREFIX: 9a:99:57:1e
```

Comment:
is it possible to make this an argument to setup_stack?  Using an env variable is a different point-of-configuration than what's already employed (command line args).  Then maybe setup_stack could set the env variable internally if it needs to cascade down to other things?

Reply:
bin/setup_stack has too many positional arguments already so I opted for an env variable.